app/routers/sessions.py

```diff
@@ -21,15 +22,20 @@ def str_to_datetime(datetime_str: str) -> datetime:
 
 
 router = APIRouter(prefix="/sessions", tags=["Sessions"])
+logger = get_logger()
 
 
 @router.post("/", response_model=SessionResponse)
 async def create_session(session: Session):
+    logger.info(
+        f"Creating new session for user: {session.user_id} and quiz: {session.quiz_id}"
+    )
     current_session = jsonable_encoder(session)
 
     quiz = client.quiz.quizzes.find_one({"_id": current_session["quiz_id"]})
 
     if quiz is None:
+        logger.error(f"Quiz {current_session['quiz_id']} not found")
```

Comment:
logger.error(f"Quiz {current_session['quiz_id']} not found while creating session") ?